src/modules/ekf2/EKF/ekf.h

```diff
@@ -583,14 +583,15 @@ class Ekf final : public EstimatorInterface
 
 	// Variables used by the initial filter alignment
 	bool _is_first_imu_sample{true};
-	AlphaFilter<Vector3f> _accel_lpf{0.1f};	///< filtered accelerometer measurement used to align tilt (m/s/s)
-	AlphaFilter<Vector3f> _gyro_lpf{0.1f};	///< filtered gyro measurement used for alignment excessive movement check (rad/sec)
+	static constexpr float _sensor_lpf_time_constant = 0.09f;
+	AlphaFilter<Vector3f> _accel_lpf{_dt_ekf_avg, _sensor_lpf_time_constant};	///< filtered accelerometer measurement used to align tilt (m/s/s)
+	AlphaFilter<Vector3f> _gyro_lpf{_dt_ekf_avg, _sensor_lpf_time_constant};	///< filtered gyro measurement used for alignment excessive movement check (rad/sec)
```

Comment:
This is a somewhat obfuscated way of setting alpha to 0.1.
https://github.com/PX4/PX4-Autopilot/blob/fd87cd682db5360de4408aefa1b93bad1eaa1d3d/src/modules/ekf2/EKF/estimator_interface.h#L344
so it's 0.01 / (0.01 + 0.09) = 0.1

But if it makes more sense in the context e.g. the dt could change and it should keep it's weight then that makes sense.

Reply:
Well, yes and no, what matters is not the value of alpha but the time constant. I didn't want to change the current behavior so I computed the equivalent time constant but we could also round it up to 0.1s instead of that 0.09s.
And yes, what I really want is to always have the same time constant regardless of dt.

Reply:
That makes sense 👍 We Could definitely round it. I'm assuming it's pretty arbitrary since 0.1 as alpha was used all over the code.